pkg/machine/qemu/command.go

```diff
@@ -46,6 +47,24 @@ func (q *QemuCmd) SetNetwork() {
 	*q = append(*q, "-netdev", "socket,id=vlan,fd=3", "-device", "virtio-net-pci,netdev=vlan,mac=5a:94:ef:e4:0c:ee")
 }
 
+// SetNetwork adds a network device to the machine
+func (q *QemuCmd) SetUSBHostPassthrough(usbs []machine.USBConfig) {
+	if len(usbs) == 0 {
+		return
+	}
+	// Add xhci usb emulation first and then each usb device
+	*q = append(*q, "-device", "qemu-xhci")
+	for _, usb := range usbs {
+		if usb.Bus != "" && usb.DevNumber != "" {
+			dev := fmt.Sprintf("usb-host,hostbus=%s,hostaddr=%s", usb.Bus, usb.DevNumber)
+			*q = append(*q, "-device", dev)
+		} else {
+			dev := fmt.Sprintf("usb-host,vendorid=%d,productid=%d", usb.Vendor, usb.Product)
+			*q = append(*q, "-device", dev)
```

Comment:
Nit: move this line outside of the if block, and  it is only needed once.

Reply:
Ok, the change:

```diff
--- a/pkg/machine/qemu/command.go
+++ b/pkg/machine/qemu/command.go
@@ -55,13 +55,13 @@ func (q *QemuCmd) SetUSBHostPassthrough(usbs []machine.USBConfig) {
        // Add xhci usb emulation first and then each usb device
        *q = append(*q, "-device", "qemu-xhci")
        for _, usb := range usbs {
+               dev := ""
                if usb.Bus != "" && usb.DevNumber != "" {
-                       dev := fmt.Sprintf("usb-host,hostbus=%s,hostaddr=%s", usb.Bus, usb.DevNumber)
-                       *q = append(*q, "-device", dev)
+                       dev = fmt.Sprintf("usb-host,hostbus=%s,hostaddr=%s", usb.Bus, usb.DevNumber)
                } else {
-                       dev := fmt.Sprintf("usb-host,vendorid=%d,productid=%d", usb.Vendor, usb.Product)
-                       *q = append(*q, "-device", dev)
+                       dev = fmt.Sprintf("usb-host,vendorid=%d,productid=%d", usb.Vendor, usb.Product)
                }
+               *q = append(*q, "-device", dev)
```